index.bs

```diff
@@ -604,6 +613,10 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of |clientDataJSON|.
                 :   {{AuthenticatorAttestationResponse/attestationObject}}
                 ::  |attestationObject|
+            :   {{ScopedCredential/clientExtensionResults}}
```

Comment:
Sigh.  This was another part of the legacy extensions text that I'd failed to catch and correct earlier.  Hopefully this is the last of it.  I significantly rewrote the offending paragraph, in part, by deleting content that duplicated things said elsewhere.